src/radio-button.tsx

```diff
@@ -93,11 +94,18 @@ type PropsChildren = {
     children?: React.ReactNode;
     render?: undefined;
     dataAttributes?: DataAttributes;
+    ['aria-labelledby']?: string;
 };
 
-const RadioButton: React.FC<PropsRender | PropsChildren> = ({value, id, dataAttributes, ...rest}) => {
+const RadioButton: React.FC<PropsRender | PropsChildren> = ({
+    value,
+    id,
+    dataAttributes,
+    'aria-labelledby': ariaLabelledby,
+    ...rest
+}) => {
     const {disabled, selectedValue, focusableValue, select, selectNext, selectPrev} = useRadioContext();
-    const labelId = useAriaId();
+    const labelId = useAriaId(ariaLabelledby);
```

Comment:
bug2: a new `labelId` was created and the `aria-labelledby` value wasn't matching with the title's id